docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Limit Trie Logs for Bonsai
+sidebar_position: 12
+description: Enable this early access feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Limit Trie Logs for Bonsai
+## Step by Step Guide
+
+:::caution
+
+We strongly recommend reading the rest of this documentation before running these commands because your node’s configuration may require changes to these commands.
+
+:::
+
+*Targeting 24.1.2 release, but currently you must be on the Besu `main` branch for this to work*
+
+1. Update Besu config to add --Xbonsai-limit-trie-logs-enabled but don’t restart yet
+1. Stop Besu
+1. (optional) Run: 
```

Comment:
should we provide relative paths?

Reply:
In this example, I opted to use the most widely used paths, which fortunately are the same for ethstaker docs and the two most popular step by step guides: Somer Esats and CoinCashew.

I put a warning to not copy and paste, but if ppl do then it should work out of the box if they followed the popular guides.